src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -956,7 +948,7 @@ def getWorkloadArguments():
                      "AutoApproveSubscriptionSites" : {"default" : [], "type" : makeList, "assign_optional": False, 
                                         "validate" : lambda x: all([cmsname(y) for y in x])},
                      # should be Low, Normal, High
-                     "SubscriptionPriority" : {"default" : "Low", "type" : str, "assign_optional": False,
+                     "SubscriptionPriority" : {"default" : "Low", "assign_optional": False,
```

Comment:
If it is optional, default value "Low" is OK,  I have check how assign_optional handled. Thanks,